src/cloudFunction.ts

```diff
@@ -197,13 +222,16 @@ export class CloudFunction {
    * @param envVarsFile env var file path.
    * @returns map of type {KEY1:VALUE1}
    */
-  protected parseEnvVarsFile(envVarFilePath: string): KVPair {
+  protected parseEnvVarsFile(
+    envVarFilePath: string,
+    optionName = 'env_vars_file',
```

Comment:
nit: I think we just use the base name instead like `myvars.yaml must contain only key/value pair of strings. ...`

Reply:
Hmm that's true! I'll fix it.